src/Adapter/Configuration.php

```diff
@@ -75,4 +75,22 @@ public function set($key, $value)
 
         return $this;
     }
+
+    /**
+     * Return if Feature feature is active or not
+     * @return bool
+     */
+    public function featureIsActive()
```

Comment:
No, it's not used for the moment. As I was working on combinaison and feature, I make 2 functions